### PR TITLE
Add support for teleport nodes in NME

### PR DESCRIPTION
**Pull request by deltakosh**
New feature!!

you can teleport data:
image

And due to popular demand, I also added a glow on one side when you pick the other side of the warp tunnel :)

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14115/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14115/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14115/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14115/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14115/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14115/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14115/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14115/merge#BCU1XR#0

**Review by carolhmj — APPROVED** [review]

**Review by Popov72 — COMMENTED** [review]
Sorry, the PR has been merged too fast and I didn't have time to review in time!

File: packages/dev/core/src/Materials/Node/nodeMaterial.ts [hunk 1]
```diff
@@ -636,16 +647,19 @@ export class NodeMaterial extends PushMaterial {
...
+                    this._processInitializeOnLink(block, state, nodesToProcessForOtherBuildState, autoConfigure);
                 }
             }
         }
 
+        // Teleportation
+        if (node.isTeleportOut) {
+            const teleport = node as NodeMaterialTeleportOutBlock;
+            if (teleport.entryPoint) {
+                this._processInitializeOnLink(teleport.entryPoint, state, nodesToProcessForOtherBuildState, autoConfigure);
```
[the remark this answers is not in the document]
Comment (deltakosh) [reply packages/dev/core/src/Materials/Node/nodeMaterial.ts:659]: This is the same as other blocks where an output is connected to several inputs.

**Pull request merged**